src/main/sensors/acceleration.c

```diff
@@ -43,9 +43,9 @@ FAST_DATA_ZERO_INIT acc_t acc;                       // acc access functions
 
 static void applyAccelerationTrims(const flightDynamicsTrims_t *accelerationTrims)
 {
-    acc.accADC[X] -= accelerationTrims->raw[X];
-    acc.accADC[Y] -= accelerationTrims->raw[Y];
-    acc.accADC[Z] -= accelerationTrims->raw[Z];
+    acc.accADC.x -= accelerationTrims->raw[X];
```

Comment:
Eventually refactor it to vector subtraction.
(and even better, use 4x3 transformation matrix (homogenous coordinates) to align everything with single matrix multiplication